include/bluetooth/l2cap.h

```diff
@@ -257,6 +257,13 @@ struct bt_l2cap_chan_ops {
 	 *  @param status The channel status
 	 */
 	void (*status)(struct bt_l2cap_chan *chan, atomic_t *status);
+
+	/* Channel released callback
```

Comment:
Needs `@brief` here (I have a PR that fixes the rest that are missing).

Reply:
Done.